build.gradle

```diff
@@ -141,6 +141,10 @@ sonarqube {
   }
 }
 
+pitest {
+  pitestVersion = '1.4.7'
```

Comment:
line 12 has `id 'info.solidsoft.pitest' version '1.4.0'`

Reply:
plugin is latest: `1.4.0`. the tool version differs and from default `1.4.3` pulled in by plugin and dependency management we now specify latest which is `1.4.7`. similarly like pmd plugin except this one has two misleading version notions :|

Reply:
Do we need pitest plugin and it's dependencies ?